browser-test/src/applicant/applicant_prevent_duplicate_submission.test.ts

```diff
@@ -0,0 +1,51 @@
+import {test} from '../support/civiform_fixtures'
+import {
+  enableFeatureFlag,
+  loginAsAdmin,
+  loginAsTestUser,
+  logout,
+  validateAccessibility,
+  validateScreenshot,
+  waitForPageJsLoad,
+} from '../support'
+
+test.describe('Prevent Duplicate Submission', {tag: ['@northstar']}, () => {
```

Comment:
suggestion (maybe blocking): There are existing tests for the old version of this screen here: https://github.com/civiform/civiform/blob/94d7c053a191b97142834114c25f1e20792ce9bb/browser-test/src/applicant/navigation/application_navigation_flow.test.ts#L391

In general, are you trying to keep north star browser tests and older browser tests together? If so, I suggest moving the older tests to this file since that navigation flow test file is pretty overloaded at this point.

Reply:
In general, I've been keeping old and new tests together. +1 to moving the old tests to this file for better sharding. we have a few test files with a ton of tests.